seedlet/template-parts/content/content.php

```diff
@@ -11,7 +11,7 @@
 ?>
 
 <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-	<header class="entry-header default-max-width">
+	<header class="entry-header alignwide">
```

Comment:
I think this utility class needs to stay `default-max-width` because in Seedlet's case, the post header and footer are aligned to the main default-width column. 

This is what a sticky post looks like when the `alignwide` class is used:

![Screen Shot 2020-10-02 at 8 25 37 AM](https://user-images.githubusercontent.com/5375500/94940856-f193f100-0488-11eb-9d2e-8ccac62d19d4.png)